hack/ginkgo-e2e.sh

```diff
@@ -166,20 +175,41 @@ fi
 PATH=$(dirname "${e2e_test}"):"${PATH}"
 export PATH
 
-# Choose the program to execute.
-program=("${ginkgo}")
-if [[ "${E2E_TEST_DEBUG_TOOL}" == "delve" ]]; then
-  program=("dlv" "exec")
-elif [[ "${E2E_TEST_DEBUG_TOOL}" == "gdb" ]]; then
-  program=("gdb")
+# Choose the program to execute and additional arguments for it.
+#
+# Note that the e2e.test binary must have been built with "make DBG=1"
+# when using debuggers, otherwise debug information gets stripped.
+case "${E2E_TEST_DEBUG_TOOL:-ginkgo}" in
+  ginkgo)
+    program=("${ginkgo}" "${ginkgo_cli_args[@]}")
```

Comment:
`ginkgo_cli_args` not found in this file, how are tests passing, shouldn't this error out on an unset var?
```suggestion
    program=("${ginkgo}")
```

Reply:
Right, another leftover. Removed.